client/js/bar.js

```diff
@@ -0,0 +1,35 @@
+/* constructs bar graph from scratch */
+
+const svg = document.createElementNS('http://www.w3.org/2000/svg', 'svg');
+svg.setAttribute('viewBox', '0 0 100 100'); // change height in css
+document.getElementById('graph').appendChild(svg);
```

Comment:
This should go below the file consts below.

Reply:
new changes implement this. Is it good to put it after file consts because all file consts pertain to the svg set up?

Reply:
Yes. Consider file const's to be parameters to the file that a developer will want to toggle without reading anything else. The only thing that should go above file const's are imports.